service/controller/chartconfig/v5/resource/tiller/create.go

```diff
@@ -0,0 +1,20 @@
+package tiller
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	r.logger.LogCtx(ctx, "level", "debug", "message", "ensuring tiller is installed")
+
+	err := r.helmClient.EnsureTillerInstalled(ctx)
```

Comment:
Calling it once and for all is already a huge improvement. We still have to improve the internals of this thing as the PMs show.

Reply:
Totally, we still need to improve helmclient. I just want to get chart-operator "done done" for app-catalog. So I wanted to get this in.